support-config/src/main/resources/touchpoint.PROD.conf

```diff
@@ -1,10 +1,14 @@
 touchpoint.backend.environments {
   PROD {
     stripe {
-      AUD {
```

Comment:
Discovered post-deploy:
This change needs to be reflected in s3 in the support-workers.private.conf, to avoid breaking recurring in AU